Dockerfile-debian.template

```diff
@@ -25,8 +25,11 @@ RUN set -ex \
 		libssl1.0-dev \
 		ruby \
 	' \
+	&& runtimeDeps=' \
+	    libjemalloc-dev
+	' \
```

Comment:
I had to add it as a separate set of dependencies, because `buildDeps` get purged after the build is finished.